src/Rpc/NodeGuardService.cs

```diff
@@ -871,7 +886,43 @@ private bool ValidateTargetBalance(LiquidityRule rule)
         return true;
     }
 
-    public override async Task<GetAvailableUtxosResponse> GetAvailableUtxos(GetAvailableUtxosRequest request, ServerCallContext context)
+    public override async Task<GetUtxosResponse> GetUtxos(GetUtxosRequest request, ServerCallContext context)
```

Comment:
Request is not used, so this is in fact, `GetAllUTXOs`

Reply:
created issue https://github.com/Elenpay/NodeGuard/issues/382